packages/liveblocks-core/src/lib/position.ts

```diff
@@ -1,133 +1,340 @@
-export const min = 32;
-export const max = 126;
+/**
+ * Positions, aka the Pos type, are efficient encodings of "positions" in
```

Comment:
this is excellent documentation! thank you!

Reply:
Awesome documentation!